src/Raven.Server/Web/Authentication/AdminApiKeysHandler.cs

```diff
@@ -36,8 +37,8 @@ public class AdminApiKeysHandler : RequestHandler
                 }
 
                 var apiKey = JsonDeserializationServer.ApiKeyDefinition(apiKeyJson);
-                await ServerStore.PutValueInClusterAsync(new PutApiKeyCommand(Constants.ApiKeys.Prefix + name, apiKey));
-
+                var res = await ServerStore.PutValueInClusterAsync(new PutApiKeyCommand(Constants.ApiKeys.Prefix + name, apiKey));
+                await ServerStore.Cluster.WaitForIndexNotification(res.Etag);
```

Comment:
Need to add validation here that user cannot create API Keys starting with `Raven/`